godot-macros/src/lib.rs

```diff
@@ -344,18 +344,28 @@ use crate::util::{bail, ident, KvParser};
 ///
 /// ## Editor plugins
 ///
-/// If you annotate a class with `#[class(editor_plugin)]`, it will be turned into an editor plugin. The
-/// class must then inherit from `EditorPlugin`, and an instance of that class will be automatically added
+/// Classes inheriting `EditorPlugin` will be automatically instantiated and added
 /// to the editor when launched.
```

Comment:
This is currently only true for directly inheriting `EditorPlugin`, I think. Eventually (latest when we allow Rust-Rust inheritance), we might want to provide a mechanism that works transitively.

Reply:
Ideally it would be true even in such a case :thinking:. Currently we register classes on runtime and gather information if class is EditorPlugin or not on compile time; we can always switch it to `Inherits::<...>`